.github/workflows/pull-request.yml

```
@@ -279,20 +279,44 @@
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    name: iOS Build and Relaese
     runs-on: macos-latest
-    needs: [Flutter-Testing]
+    needs: Flutter-Testing
     steps:
       - uses: actions/checkout@v4
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: "3.29.3"
+          flutter-version: "3.32.8"
           channel: "stable" # or: 'beta', 'dev' or 'master'
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)